apis/logging/v1/cluster_log_forwarder_types.go

```diff
@@ -232,6 +232,11 @@ type PipelineSpec struct {
 	// +kubebuilder:validation:Enum:=json
 	// +optional
 	Parse string `json:"parse,omitempty"`
+
+	// EnableMultilineErrorDetection enables multiline error detection of container logs
+	//
+	// +optional
+	EnableMultilineErrorDetection bool `json:"enableMultilineErrorDetection,omitempty"`
```

Comment:
Suggest "detectMultilineErrors: true/false" - sorry for the nit, but I think it's important to keep the API as brief as possible without sacrificing clarity.